app/Http/Transformers/PhoenixGalleryTransformer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Rogue\Http\Transformers;
+
+use Rogue\Models\Post;
+use League\Fractal\TransformerAbstract;
+
+class PhoenixGalleryTransformer extends TransformerAbstract
+{
+    /**
+     * Transform resource data.
+     *
+     * @param \Rogue\Models\Photo $photo
+     * @return array
+     */
+    public function transform(Post $post)
+    {
+        $signup = $post->signup;
+        $result = [
+            'id' => $post->postable_id,
+            'status' => $post->status,
+            'caption' => $post->content->caption,
+            'uri' => $post->content->file_url,
```

Comment:
@katiecrane One of these points to the admin page for a repotback item. we provide a little administration link per reportback when admins are viewing the page. we don't have the equivalent in Rogue yet, so I _think_ you can leave it out with a note we need to add it in when it is created. And then we can leave the media properties the same since that points to the actual file url.